engine/api/warning/warning_project.go

```diff
@@ -0,0 +1,138 @@
+package warning
```

Comment:
maybe we should make a package warningproject

Reply:
we will see after :)